msteams-platform/bots/how-to/conversations/command-bot-in-teams.md

```diff
@@ -0,0 +1,335 @@
+
+:::image type="content" source="../../../assets/images/command-bot-teams/commandbot-installation.png" alt-text="installation option selection":::
+
+For more install options, see [configure default install options](../../../concepts/deploy-and-publish/apps-publish-overview.md#configure-default-install-options).
+
+## Command and response
+
+The TeamsFx command and response bots are built using the Bot Framework SDK. The Bot Framework SDK provides built-in message handler to handle the incoming message activity, which requires you to understand the concept of Bot Framework such as the event-driven conversation model. TeamsFx SDK provides command-response abstraction layer to let the users focus on handling the command request according to the business need, without learning the Bot Framework SDK.
+
+TeamsFx SDK pulls Bot Framework middleware to handle the integration with the underlying activity handlers. If the received message text matches the command pattern provided in a `TeamsFxBotCommandHandler` instance, the middleware handles the incoming message activity and invokes the corresponding `handlerCommandReceived` function. After this process, the middleware calls `context.sendActivity` to send the command response returned from the `handlerCommandReceived` function to the user.
+
+## Customize initialization
+
+You can initialize with your own adapter or customize after initialization.
+
+```js(ts)
+
+   // Create your own adapter
+   const adapter = new BotFrameworkAdapter(...);
+
+   // Customize your adapter, e.g., error handling
+   adapter.onTurnError = ...
+
+   const bot = new ConversationBot({
+       // use your own adapter
+       adapter: adapter;
+       ...
+   });
+
+   // Or, customize later
+   bot.adapter.onTurnError = ...
+
+```
+
+## Add command and response
+
+You can perform the following steps to add command and responses:
+
+<br>
+
+<details>
+
+<summary><b>1. Add a command definition in manifest</b></summary>
+
+You can edit the manifest template file `templates\appPackage\manifest.template.json` to include:
+
+* The command title that user types in the message compose area to trigger the command.
+* The description for the command.
+
+  :::image type="content" source="../../../assets/images/command-bot-teams/commandbot-add-command-definition.png" alt-text="add a command definition in manifest code sample" lightbox="../../../assets/images/command-bot-teams/commandbot-add-command-definition.png":::
+
+<br>
+</details>
+
+<details>
+
+<summary><b>2. Respond with an Adaptive Card</b></summary>
+
+You can build your response data in text format, or perform the following steps to use an Adaptive Card to apply rich content in Teams:
+
+* Prepare your Adaptive Card content in a JSON file such as `myCard.json` under the `bot/adaptiveCards` folder. Following is a sample Adaptive Card with JSON payload:
```

Comment:
How about change the adaptive card naming to keep consistent with readme:  `myCard.json` -> `doSomethingCommandResponse.json`

Reply:
This has been updated as per readme.